js/pdf_exporter.d.ts

```diff
@@ -0,0 +1,67 @@
+import dxDataGrid from './ui/data_grid';
+
+/**
+* @docid PdfExportBaseProps
+* @namespace DevExpress.pdfExporter
+* @type object
+* @hidden
+*/
+export interface PdfExportBaseProps {
```

Comment:
imho we should not split interface into Base and DataGrid props.
It's easy to split it when we will add Pivot props

Reply:
I fully agree with you. We don't want to get ahead of ourselves here. I merged two interfaces into one. https://github.com/DevExpress/DevExtreme/pull/14787/commits/27a4e5e7240e73d918f2db63bf62930dec56dfa3